mlir-tensorrt/python/bindings/Runtime/RuntimePyBind.cpp

```diff
@@ -775,7 +775,18 @@ PYBIND11_MODULE(_api, m) {
             THROW_IF_MTRT_ERROR(s);
           },
           py::arg("device_memref"), py::arg("existing_host_memref"),
-          py::arg("stream") = py::none());
+          py::arg("stream") = py::none())
+      .def(
+          "report_allocated_memory",
+          [](PyRuntimeClient &self) {
+            int64_t totalGpuMemory;
+            int64_t totalCpuMemory;
+            MTRT_Status s = mtrtReportAllocatedMemory(self, &totalCpuMemory, &totalGpuMemory);
+            THROW_IF_MTRT_ERROR(s);
+            py::object namedtuple = py::module::import("collections").attr("namedtuple");
+            py::object MemoryUsage = namedtuple("MemoryUsage", "cpu_memory gpu_memory");
```

Comment:
You'll need to update the stubs so users can see this type information in the IDE.